server/app/controllers/UserController.ts

```diff
@@ -40,4 +41,14 @@ export class UserController extends BaseHttpController {
       return this.json(err.message, 400);
     }
   }
+
+  @httpPost('/email')
```

Comment:
im wondering if we should maybe just create a new controller, so the path can just be /email

Reply:
I was thinking the same, I can create a controller for the system such as /system/email. What do you think?

Reply:
ya that sounds good, i dont think there should be anything email related in the user controller